litellm/tests/test_router_caching.py

```diff
@@ -134,6 +134,56 @@ async def test_acompletion_caching_on_router():
         traceback.print_exc()
         pytest.fail(f"Error occurred: {e}")
 
+@pytest.mark.asyncio
+async def test_completion_caching_on_router():
+    # tests completion + caching on router
+    try:
+        litellm.set_verbose = True
+        model_list = [
+            {
+                "model_name": "gpt-3.5-turbo",
+                "litellm_params": {
+                    "model": "gpt-3.5-turbo",
+                    "api_key": os.getenv("OPENAI_API_KEY"),
+                },
+                "tpm": 1000,
+                "rpm": 1,
+            },
+        ]
+        
+        messages = [
+            {"role": "user", "content": f"write a one sentence poem {time.time()}?"}
+        ]
+        router = Router(
+            model_list=model_list,
+            redis_host=os.environ["REDIS_HOST"],
+            redis_password=os.environ["REDIS_PASSWORD"],
+            redis_port=os.environ["REDIS_PORT"],
+            cache_responses=True,
+            timeout=30,
+            routing_strategy_args={"ttl": 10},
```

Comment:
Hi @krrishdholakia 
followed the same as `latency-based-routing`.
default TTL value is 1min(60secs), for this test case alone here we set it for only 10sec.